### DISCUSSION TITLE
The icon used for branch deletion doesn't make sense

### DISCUSSION THREAD

**Discussion by t184256** [community/community#8070] [Pull Requests]
Consider this example: https://github.com/p11-glue/p11-kit/pull/367

If the time goes upwards, then merge icon does indeed look like a zoom-in on a merge commit, but the deletion icon looks nothing like a branch deletion icon. It's, arguably, the opposite of it, a branching-out icon.

If the time goes downwards, it makes even less sense: the deletion icon looks like a merging icon and the merge icon looks extra puzzling.

**Comment by ca-d** [comment]
What's the example?

**Reply by t184256** [reply]
https://github.com/p11-glue/p11-kit/pull/367
